app/controllers/teams_controller.rb

```diff
@@ -1,14 +1,10 @@
 class TeamsController < ApplicationController
-  before_action :set_team, only: [:show, :edit, :update, :destroy]
-  load_and_authorize_resource
-  before_action :set_team, only: [:show, :edit, :update, :destroy, :assign_ownership, :delete_membership, :delete_ownership]
-  # before_action :set_user, only: [:assign_ownership, :delete_membership, :delete_ownership]
-  before_action :owners_include_user, only: [:assign_ownership, :delete_membership, :delete_ownership]
+  before_action :set_team, only: [:show, :edit, :update, :destroy, :assign_ownership, :delete_membership, :delete_ownership, :perform_action_on_multiple_members]
   before_action :authenticate_user!, only: [:new, :create, :edit, :update, :destroy]
-  load_and_authorize_resource :team
 
   # GET /teams
   def index
+    @teams = Team.all
```

Comment:
Please either use something like `accessible_by` here or re-introduce the deleted `load_and_authorize_resource :team` line. Otherwise, the read abilities (for private teams) won’t work properly.